Lib/getpass.py

```diff
@@ -108,9 +119,16 @@ def win_getpass(prompt='Password: ', stream=None):
         if c == '\003':
             raise KeyboardInterrupt
         if c == '\b':
+            if echochar and pw:
+                for _ in echochar:
+                    msvcrt.putwch('\b')
```

Comment:
Is there a function to write multiple characters for msvcrt?

Reply:
[It seems](https://docs.python.org/dev/library/msvcrt.html) that there is not